golos.publication/golos.publication.cpp

```diff
@@ -82,6 +83,7 @@ struct posting_params_setter: set_params_visitor<posting_state> {
 
 void publication::create_message(structures::mssgid message_id,
                               structures::mssgid parent_id,
+                              uint64_t parent_recid,
```

Comment:
this argument is rarely used, maybe we should put it to the end of the list and make optional or extension-like.
it's not very important for now, looks like other `create_message` arguments can be implemented better too